source/operations/checklists/hardware.rst

```diff
@@ -8,4 +8,233 @@ Hardware Checklist
+      * - ``count=80000``
+        - Write up to 80000 blocks of data
+      * - ``oflag=direct``
+        - Use direct I/O to write to avoid data from caching
+      * - ``conv=fdatasync``
+        - Physically write output file data before finishing
+      * - ``> dd-write-drive1.txt``
+        - Write the contents of the operation's output to ``dd-write-drive1.txt`` in the current working directory
+
+   The operation returns the number of files written, total size written in bytes, the total length of time for the operation (in seconds), and the speed of the writing in some order of bytes per second.
+
+#. Test the drive's performance during read operations
+
+   .. code-block::
+      :class: copyable
+
+      dd if=/mnt/driveN/testfile of=/dev/null bs=128k iflag=direct > dd-read-drive1.txt
+
+   Replace ``driveN`` with the path for the disk you are testing.
+
+   .. list-table::
+      :widths: auto
+      :width: 100%
+
+      * - ``dd``
+        - The command to copy and paste data
+      * - ``if=/mnt/driveN/testfile``
+        - Read from ``/mnt/driveN/testfile``; replace with the path to the file to use for testing the drive's read performance
+      * - ``of=/dev/null``
+        - Write to ``/dev/null``, a virtual file that does not persist after the operation completes
+      * - ``bs=128k``
+        - Write up to 128,000 bytes at a time
+      * - ``count=80000``
+        - Write up to 80000 blocks of data
+      * - ``iflag=direct``
+        - Use direct I/O to read and avoid data from caching
+      * - ``> dd-read-drive1.txt``
+        - Write the contents of the operation's output to ``dd-read-drive1.txt`` in the current working directory
+
+   Use a sufficiently sized file that mimics the primary use case for your deployment to get accurate read test results.
+   
+   The following guidelines may help during performance testing:
+
+   - Small files: < 128KB
+   - Normal files: 128KB – 1GB
+   - Large files: > 1GB
+
+   You can use the ``head`` command to create a file to use.
+   The following command example creates a 10 Gigabyte file called ``testfile``.
+
+   .. code-block:: shell
+      :class: copyable
+
+      head -c 10G </dev/urandom > testfile
+
+   The operation returns the number of files read, total size read in bytes, the total length of time for the operation (in seconds), and the speed of the reading in bytes per second.
+
+#. IO Controller test
+   
+   Use `IOzone <http://iozone.org/>`__ to test the input/output controller and all drives in combination.
+   Document the performance numbers for each server in your deployment.
+
+   .. code-block:: shell
+      :class: copyable
+
+      iozone -s 1g -r 4m -i 0 -i 1 -i 2 -I -t 160 -F /mnt/sdb1/tmpfile.{1..16} /mnt/sdc1/tmpfile.{1..16} /mnt/sdd1/tmpfile.{1..16} /mnt/sde1/tmpfile.{1..16} /mnt/sdf1/tmpfile.{1..16} /mnt/sdg1/tmpfile.{1..16} /mnt/sdh1/tmpfile.{1..16} /mnt/sdi1/tmpfile.{1..16} /mnt/sdj1/tmpfile.{1..16} /mnt/sdk1/tmpfile.{1..16} > iozone.txt
```

Comment:
Same comment as above for dd, iozone/fio can be tabled in liew of just mc support drive.

Reply:
As above, split out iozone into a "Third Part Diagnostics" section.